services/CertificateGeneration/DockerFile

```diff
@@ -0,0 +1,21 @@
+FROM microsoft/dotnet:2.0-sdk AS build

```

Comment:
I think this should be the .netcore runtime, not SDK. The SDK is going to cause the docker image to be huge. https://hub.docker.com/r/microsoft/dotnet/